src/components/Popover/index.tsx

```diff
@@ -117,17 +122,36 @@ export default function Popover({
   const updateCallback = useCallback(() => {
     update && update()
   }, [update])
-  useInterval(updateCallback, show ? 100 : null)
+  useInterval(updateCallback, show ? 200 : null)
```

Comment:
Why are you updating this interval?

```suggestion
  useInterval(updateCallback, show ? 100 : null)
```

Reply:
undone